viewer/packages/cad-model/src/batching/MultiBufferBatchingManager.ts

```diff
@@ -323,6 +323,7 @@ export class MultiBufferBatchingManager implements DrawCallBatchingManager {
     const instancedMesh = new InstancedMesh(instanceBufferGeometry, material, 0);
     instancedMesh.visible = false;
     instancedMesh.frustumCulled = false;
+    instancedMesh.boundingSphere = new Sphere(); // Creates a dummy bounding sphere to avoid a calculated sphere on first render. The THREEjs calculated sphere is never updated after the first calculate, so the calculated sphere is misleading after we add data to the batch)
```

Comment:
Can we shorten this comment? :sweat_smile: Something along the lines of 
```suggestion
    instancedMesh.boundingSphere = new Sphere(); // Unused, to avoid a calculated sphere on first render
```